taxi/forms.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreateForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "username",
+            "first_name",
+            "last_name",
+            "license_number",
+            "email",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number"]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8 or not re.match(r"^[A-Z]{3}\d{5}$",
+                                                    license_number):
+            raise forms.ValidationError(
+                "License number must be 8 characters long,"
+                " the first 3 uppercase letters and the last 5 digits.")
```

Comment:
The regular expression pattern used in `clean_license_number` is incorrect. It should be `^[A-Z]{3}\d{5}$` to match exactly 3 uppercase letters followed by 5 digits. The current pattern might not work as intended due to incorrect escaping of the backslash.